eng/testing/mobile/WasmRunnerTemplate.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+EXECUTION_DIR=$(dirname $0)
+TEST_NAME=$1
+JS_ENGINE=$2
+JS_ENGINE_ARGS=$3
+TEST_FILTER=$4
+XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
+
+if [ -z ${2+x} ]; then
+	JS_ENGINE=V8
+fi
+
+if [ -z ${4+x} ]; then
+    TEST_FILTER="-notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing"
+fi
```

Comment:
a nit: in some places, we are using the default values syntax:
```suggestion
EXECUTION_DIR="$(dirname $0)"
TEST_NAME="$1"
JS_ENGINE="${2:-V8}"
JS_ENGINE_ARGS="$3"
TEST_FILTER="${4:--notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing}"
XHARNESS_OUT="$EXECUTION_DIR/xharness-output"
```

Reply:
cool. will this work on all shells that we care about?

Reply:
Yup, it is a POSIX shell syntax (which is a subset of bash; first line of this file has called out bash interpreter).
Confirmed with https://www.shellcheck.net/. :)

Reply:
Thanks a lot :)